backend/app/poller.py

```
@@ -1731,10 +1729,124 @@ async def run_room_setup(room_info, loop):
+                                                                
+                                                                async with datapackage_locks[chk]:
+                                                                    still_missing = await loop.run_in_executor(None, db_get_missing_checksums, [chk])
+                                                                    if not still_missing:
+                                                                        continue
+                                                                        
+                                                                    current_game_entries = []
+                                                                    seen_ids = set()
+                                                                    
+                                                                    # 1. Items
+                                                                    for n, eid in game_data.get('item_name_to_id', {}).items():
+                                                                        if ('item', eid) in seen_ids: continue
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)